domain/src/test/java/org/oppia/android/domain/translation/TranslationControllerTest.kt

```diff
@@ -1371,6 +1371,183 @@ class TranslationControllerTest {
     assertThat(selection.selectedLanguage).isEqualTo(ENGLISH)
   }
 
+  @Test
+  fun testGetAudioLanguagePreselection_noPreviousSelections_returnsSystemLanguageAsPreselection() {
+    forceDefaultLocale(Locale.ENGLISH)
+
+    val preselectionProvider = translationController.getAudioLanguagePreselection(PROFILE_ID_0)
+
+    // English is returned since it is the configured system language.
+    val language = monitorFactory.waitForNextSuccessfulResult(preselectionProvider)
+    assertThat(language).isEqualTo(ENGLISH)
+  }
+
+  @Test
+  fun testGetAudioLanguagePreselection_IndiaDefaultLocale_returnsHindiLanguageAsPreselection() {
```

Comment:
```suggestion
  fun testGetAudioLanguagePreselection_indiaDefaultLocale_returnsHindiLanguageAsPreselection() {
```

Nit on snake case capitalization.

Reply:
Fixed.